From c07888b06a5ad430e137f3b48dba5c0cb9ea7649 Mon Sep 17 00:00:00 2001
From: Chenxiong Qi <cqi@redhat.com>
Date: Thu, 26 Oct 2023 13:12:33 +0800
Subject: [PATCH] Increase counter as numeric rather than string

As a result, rather than extra-src-0111.tar, the file name will be
extra-src-0.tar, extra-src-1.tar, extra-src-2.tar, etc.

Signed-off-by: Chenxiong Qi <cqi@redhat.com>
---
 BuildSourceImage.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/BuildSourceImage.sh b/BuildSourceImage.sh
index 9a1e18c..ace5232 100755
--- a/BuildSourceImage.sh
+++ b/BuildSourceImage.sh
@@ -1039,7 +1039,7 @@ sourcedriver_extra_src_dir() {
         _info "adding extra source directory $extra_src_dir"
         _debug "$self: writing to $out_dir and $manifest_dir"
         tarname="extra-src-${counter}.tar"
-        counter+=1
+        ((counter+=1))
         _tar -C "${extra_src_dir}" \
             --sort=name --mtime=@0 --owner=0 --group=0 --mode='a+rw' --no-xattrs --no-selinux --no-acls \
             -cf "${out_dir}/${tarname}" .
-- 
2.41.0

